### PR TITLE
Fix copy of the chat error

### PR DESCRIPTION
**Pull request by pkukielka**
Fixes https://github.com/sourcegraph/cody-issues/issues/260

## Changes

image

## Test plan

Open Cody, disable network connection, type any chat question.

### CODE REVIEW

File: src/main/resources/CodyBundle.properties [hunk 1]
```diff
@@ -45,11 +45,11 @@ chat.rate-limit-error.explain=\
   </html>
 chat.general-error=\
   <html>\
-   <b> Cody encountered an error when processing your message:</b><br>\
-    <div marginwidth=10" marginheight="10">&#9888;  {0}</div>\
-    <div>{1}</div>\
+    <b>&#9888; Error performing this action</b><br><br>\
+    Please retry sending your message. If you tried to run a command, try it again.<br>\
+    If the issue persists, please <a href="{0}">create a support ticket</a>.<br><br>\
```
Comment (mkondratek) [inline src/main/resources/CodyBundle.properties:50]: pls double check that the link is working properly

**Review by mkondratek — APPROVED** [review]
lgtm

File: src/main/resources/CodyBundle.properties (hunk 1 above)
Comment (mkondratek) [reply src/main/resources/CodyBundle.properties:50]: for me it generated a link like this:

```
https://github.com/sourcegraph/jetbrains/issues/new?template=bug_report.yml&labels=bug,team/jetbrains&projects=sourcegraph/381&title=bug%3A+java.util.concurrent.ExecutionException%3A+org.eclipse.lsp4j.jsonrpc.ResponseErrorException%3A+Request+chat%2Frestore+failed+with+m...&version=IDE+version%3A+%60%60%60IC-221.5080.210%60%60%60&logs=Stacktrace%3A%0A%60%60%60text%0Ajava.util.concurrent.ExecutionException%3A+org.eclipse.lsp4j.jsonrpc.ResponseErrorException%3A+Request+chat%2Frestore+failed+with+message%3A+No+default+chat+model+found%0A%09at+java.base%2Fjava.util.concurrent.CompletableFuture.reportGet%28CompletableFuture.java%3A395%29%0A%09at+java.base%2Fjava.util.concurrent.CompletableFuture.get%28CompletableFuture.java%3A1999%29%0A%09at+com.sourcegraph.cody.chat.AgentChatSession.submitMessageToAgent%24lambda%240%28AgentChatSession.kt%3A122%29%0A%09at+com.sourcegraph.cody.agent.CodyAgentService%24Companion%24withAgent%242%241%24task%241.invokeSuspend%28CodyAgentService.kt%3A251%29%0A%09at+com.sourcegraph.cody.agent.CodyAgentService%24Companion%24withAgent%242%241%24task%241.invoke%28CodyAgentService.kt%29%0A%09at+com.sourcegraph.cody.agent.CodyAgentService%24Companion%24withAgent%242%241%24task%241.invoke%28CodyAgentService.kt%29%0A%09at+com.sourcegraph.cody.agent.CodyAgentService%24Companion.coWithAgent%28CodyAgentService.kt%3A307%29%0A%09at+com.sourcegraph.cody.agent.CodyAgentService%24Companion%24withAgent%242%241.invokeSuspend%28CodyAgentService.kt%3A256%29%0A%09at+kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith%28ContinuationImpl.kt%3A33%29%0A%09at+kotlinx.coroutines.DispatchedTask.run%28DispatchedTask.kt%3A106%29%0A%09at+kotlinx.coroutines.EventLoopImplBase.processNextEvent%28EventLoop.common.kt%3A274%29%0A%09at+kotlinx.coroutines.BlockingCoroutine.joinBlocking%28Builders.kt%3A85%29%0A%09at+kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking%28Builders.kt%3A59%29%0A%09at+kotlinx.coroutines.BuildersKt.runBlocking%28Unknown+Source%29%0A%09at+kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking%24default%28Builders.kt%3A38%29%0A%09at+kotlinx.coroutines.BuildersKt.runBlocking%24default%28Unknown+Source%29%0A%09at+com.sourcegraph.cody.agent.CodyAgentService%24Companion.withAgent%24lambda%242%28CodyAgentService.kt%3A248%29%0A%09at+com.intellij.openapi.application.impl.ApplicationImpl%241.run%28ApplicationImpl.java%3A295%29%0A%09at+java.base%2Fjava.util.concurrent.Executors%24RunnableAdapter.call%28Executors.java%3A515%29%0A%09at+java.base%2Fjava.util.concurrent.FutureTask.run%28FutureTask.java%3A264%29%0A%09at+java.base%2Fjava.util.concurrent.ThreadPoolExecutor.runWorker%28ThreadPoolExecutor.java%3A1128%29%0A%09at+java.base%2Fjava.util.concurrent.ThreadPoolExecutor%24Worker.run%28ThreadPoolExecutor.java%3A628%29%0A%09at+java.base%2Fjava.util.concurrent.Executors%24PrivilegedThreadFactory%241%241.run%28Executors.java%3A668%29%0A%09at+java.base%2Fjava.util.concurrent.Executors%24PrivilegedThreadFactory%241%241.run%28Executors.java%3A665%29%0A%09at+java.base%2Fjava.security.AccessController.doPrivileged%28Native+Method%29%0A%09at+java.base%2Fjava.util.concurrent.Executors%24PrivilegedThreadFactory%241.run%28Executors.java%3A665%29%0A%09at+java.base%2Fjava.lang.Thread.run%28Thread.java%3A829%29%0ACaused+by%3A+org.eclipse.lsp4j.jsonrpc.ResponseErrorException%3A+Request+chat%2Frestore+failed+with+message%3A+No+default+chat+model+found%0A%09at+org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleResponse%28RemoteEndpoint.java%3A209%29%0A%09at+org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume%28RemoteEndpoint.java%3A193%29%0A%09at+org.eclipse.lsp4j.jsonrpc.TracingMessageConsumer.consume%28TracingMessageConsumer.java%3A114%29%0A%09at+org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage%28StreamMessageProducer.java%3A194%29%0A%09at+org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen%28StreamMessageProducer.java%3A94%29%0A%09at+org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run%28ConcurrentMessageProcessor.java%3A113%29%0A%09at+java.base%2Fjava.util.concurrent.Executors%24RunnableAdapter.call%28Executors.java%3A515%29%0A%09at+java.base%2Fjava.util.concurrent.FutureTask.run%28FutureTask.java%3A264%29%0A%09at+java.base%2Fjava.util.concurrent.ThreadPoolExecutor.runWorker%28ThreadPoolExecutor.java%3A1128%29%0A%09at+java.base%2Fjava.util.concurrent.ThreadPoolExecutor%24Worker.run%28ThreadPoolExecutor.java%3A628%29%0A%09...+1+more%0A%0A%60%60%60%0AAdditional+info%3A+%60%60%60org.eclipse.lsp4j.jsonrpc.ResponseErrorException%3A+Request+chat%2Frestore+failed+with+message%3A+No+default+chat+model+found%60%60%60
```

I wonder - why doesn't it feel Cody Version and IDE Information automatically? 🤔

Comment (pkukielka) [reply src/main/resources/CodyBundle.properties:50]: I have not changed the way link is generated.
How it should look like in your opinion?

Comment (mkondratek) [reply src/main/resources/CodyBundle.properties:50]: Issue opened: https://github.com/sourcegraph/jetbrains/issues/1592

**Pull request merged**